### DISCUSSION TITLE
Welcome to micropython-ota-updater Discussions!

### DISCUSSION THREAD

**Discussion by rdehuyss** [rdehuyss/micropython-ota-updater#12] [General]
## 👋 Welcome!
  I'm using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by sujoy743175** [comment]
Hi there.. its sujoy . im very new to this software thing and trying to gain some knowledge ..wanted to see how this OTA thing works .. Accidentally copied your repository. I will delete as soon I come to know how these things work.. Thank you

**Comment by TomBurg** [comment]
Your OTA-updater project is an excellent example from clearly a skilled programmer.
I attempted to understand your code more than a year ago but was still too new to Micropython to get most of the nuances.
Today I was more successful and was able to install and run the updater on a Huzzah32 from Adafruit.
I am still reading your article on medium.com from 2018. Do you have a newer version since you updated the code in 2020?

I still need to understand where I place my project code on my github repository. But hopefully with a little more sweat and tears I will follow your logic.

Thank you for sharing!
